algebras/algebra/src/main/scala/endpoints4s/algebra/ChunkedEntities.scala

```diff
@@ -77,24 +94,63 @@ trait ChunkedEntities extends EndpointsWithCustomErrors {
   *
   * @group algebras
   */
-trait ChunkedJsonEntities extends ChunkedEntities with JsonCodecs {
+trait ChunkedJsonEntities extends ChunkedJsonRequestEntities with ChunkedJsonResponseEntities
+
+/** @group algebras */
+trait ChunkedJsonRequestEntities extends ChunkedRequestEntities with JsonCodecs with Framing {
+
+  @deprecated(
+    "Use jsonChunksRequest[A](framing: RequestFraming) instead to explicitly provide chunk framing",
+    "1.7.0"
+  )
+  def jsonChunksRequest[A](implicit
+      codec: JsonCodec[A]
+  ): RequestEntity[Chunks[A]]
 
   /** A request entity carrying chunks of JSON values
     *
     * @tparam A Type of values serialized into JSON
     * @group operations
     */
-  def jsonChunksRequest[A](implicit
+  def jsonChunksRequest[A](framing: Framing)(implicit
       codec: JsonCodec[A]
   ): RequestEntity[Chunks[A]]
+}
+
+trait ChunkedJsonResponseEntities extends ChunkedResponseEntities with JsonCodecs with Framing {
```

Comment:
This one too needs to be labelled `@group algebras`:
```suggestion
/** @group algebras */
trait ChunkedJsonResponseEntities extends ChunkedResponseEntities with JsonCodecs with Framing {
```